pom.xml

```diff
@@ -280,6 +288,78 @@
+                        </goals>
+                        <configuration>
+                            <url>https://raw.githubusercontent.com/redhat-developer/app-services-operator/main/olm/olm-catalog/rhoas-operator/${rhoas-model.version}/manifests/rhoas-operator.cloudservicesrequests.crd.yaml</url>
+                            <outputFileName>rhoas-operator.cloudservicesrequests.crd.yaml</outputFileName>
+                            <outputDirectory>${project.build.directory}/manifests</outputDirectory>
+                        </configuration>
+                    </execution>
+                    <execution>
+                        <id>generate-sources-rhoas-kafkaconnections</id>
+                        <phase>generate-sources</phase>
+                        <goals>
+                            <goal>wget</goal>
+                        </goals>
+                        <configuration>
+                            <url>https://raw.githubusercontent.com/redhat-developer/app-services-operator/main/olm/olm-catalog/rhoas-operator/${rhoas-model.version}/manifests/rhoas-operator.kafkaconnections.crd.yaml</url>
+                            <outputFileName>rhoas-operator.kafkaconnections.crd.yaml</outputFileName>
+                            <outputDirectory>${project.build.directory}/manifests</outputDirectory>
+                        </configuration>
+                    </execution>
+                    <execution>
+                        <id>generate-sources-rhoas-serviceregistryconnections</id>
+                        <phase>generate-sources</phase>
+                        <goals>
+                            <goal>wget</goal>
+                        </goals>
+                        <configuration>
+                            <url>https://raw.githubusercontent.com/redhat-developer/app-services-operator/main/olm/olm-catalog/rhoas-operator/${rhoas-model.version}/manifests/rhoas-operator.serviceregistryconnections.crd.yaml</url>
+                            <outputFileName>rhoas-operator.serviceregistryconnections.crd.yaml</outputFileName>
+                            <outputDirectory>${project.build.directory}/manifests</outputDirectory>
+                        </configuration>
+                    </execution>
+                </executions>
+            </plugin>
+            <plugin>
+                <groupId>io.fabric8</groupId>
+                <artifactId>java-generator-maven-plugin</artifactId>
+                <version>${fabric8.version}</version>
+                <executions>
+                  <execution>
+                    <phase>generate-sources</phase>
+                    <goals>
+                      <goal>generate</goal>
+                    </goals>
+                  </execution>
+                </executions>
+                <configuration>
+                  <source>${project.build.directory}/manifests</source>
+                  <extraAnnotations>true</extraAnnotations>
+                </configuration>
```

Comment:
just a little different indentation from rest of the file.

Reply:
should be fixed now 👍